automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -353,15 +351,18 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
         print('The Capsule repo URL is not provided '
               'to perform Capsule Upgrade in feature!')
         sys.exit(1)
-    version = distro_info()[1]
     details = os.environ.get('CAPSULE_SUBSCRIPTION')
+    rhev_cv, rhev_env, rhev_ak = [os.environ.get(env_var) for env_var in (
+        'CAPSULE_CV', 'CAPSULE_ENVIRONMENT', 'CAPSULE_AK')]
     if details is not None:
         cv_name, env_name, ak_name = [
             item.strip() for item in details.split(',')]
+    elif not rhev_cv or not rhev_env or not rhev_ak:
+        print('Error ! The CV, Env and AK details are not provided for Capsule'
```

Comment:
Can you please remove the space before `!`, this also applies for the next line.